server/events/plan_command_runner.go

```diff
@@ -318,6 +327,37 @@ func (p *PlanCommandRunner) updateCommitStatus(ctx *command.Context, pullStatus
 	}
 }
 
+func (p *PlanCommandRunner) updateApplyCommitStatus(ctx *command.Context, pullStatus models.PullStatus) {
+	if !p.SetAtlantisApplyCheckSuccessfulIfNoChanges {
+		return
+	}
+	var numSuccess int
+	var numErrored int
+	status := models.SuccessCommitStatus
+
+	numSuccess = pullStatus.StatusCount(models.AppliedPlanStatus) + pullStatus.StatusCount(models.PlannedNoChangesPlanStatus)
+	numErrored = pullStatus.StatusCount(models.ErroredApplyStatus)
+
+	if numErrored > 0 {
+		status = models.FailedCommitStatus
+	} else if numSuccess < len(pullStatus.Projects) {
+		// If there are plans that haven't been applied yet, we'll use a pending
+		// status.
+		status = models.PendingCommitStatus
+	}
+
+	if err := p.commitStatusUpdater.UpdateCombinedCount(
+		ctx.Pull.BaseRepo,
+		ctx.Pull,
+		status,
+		command.Apply,
```

Comment:
Instead of copying and pasting the `updateCommitStatus` function, can we simply expose the `command.Apply` argument here and then pass it in from

```
func (p *PlanCommandRunner) run(ctx *command.Context, cmd *CommentCommand) {
```